sound/soc/intel/boards/sof_maxim_common.c

```diff
@@ -9,6 +9,8 @@
 #include <uapi/sound/asound.h>
 #include "sof_maxim_common.h"
 
+#define MAX_98373_PIN_NAME 16
```

Comment:
please remove change id from commit message and fix typo "suggest" -> "suggests"

Reply:
Sure